src/import/chrome.ts

```diff
@@ -22,15 +23,15 @@ interface PositionTickInfo {
   ticks: number
 }
 
-interface CPUProfileCallFrame {
+export interface CPUProfileCallFrame {
```

Comment:
I don't think this needs to be exported
```suggestion
interface CPUProfileCallFrame {
```